apps/etterlatte-behandling/src/main/kotlin/behandling/tilgang/TilgangRoutes.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.etterlatte.behandling.tilgang
+
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import no.nav.etterlatte.libs.common.BEHANDLINGSID_CALL_PARAMETER
+import no.nav.etterlatte.libs.common.SAKID_CALL_PARAMETER
+import no.nav.etterlatte.libs.common.behandlingsId
+import no.nav.etterlatte.sak.SakService
+import no.nav.etterlatte.sak.SakServiceAdressebeskyttelse
+
+internal fun Route.tilgangRoutes(sakService: SakService, sakServiceAdressebeskyttelse: SakServiceAdressebeskyttelse) {
+    route("/tilgang") {
+        post("/person") {
+            val fnr = call.receive<String>()
+            call.respond(sakService.sjekkOmSakHarStrengtFortroligBeskyttelse(fnr))
+        }
+
+        get("/behandling/{$BEHANDLINGSID_CALL_PARAMETER}") {
```

Comment:
Denne routen vil vel bli interceptet? Det ønsker vi vel egentlig ikke 🤔

Reply:
Foreslår at vi legger den inn i restModule via en egen param for unauthenticated routes som ikke får hooken installert på seg. Så slipper vi whitelisting i hooken.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/934/commits/7a3fdf4a468abb958107b054fd5f525fa667976c